apps/docs/pages/guides/auth/auth-helpers/nextjs-server-components.mdx

```diff
@@ -220,11 +220,13 @@ export default function SupabaseListener({ accessToken }) {
   const router = useRouter()
 
   useEffect(() => {
-    supabase.auth.onAuthStateChange((event, session) => {
+    const event = supabase.auth.onAuthStateChange((event, session) => {
```

Comment:
I think we can make this a little more concise, for example:

`const { data: subscription } = supabase.auth.onAuthStateChange`

and then just `subscription.unsubscribe()`